frontend/src/views/public/postings/survey.css

```diff
@@ -11,4 +11,12 @@ the active button to be bright. */
 
 .sv_main.sv_main .sv-boolean__switch {
     background-color: var(--primary) !important;
+}
+
+.table.sv_q_matrix thead th, .table.sv_q_matrix thead td {
```

Comment:
We could specify the `position_preferences` name directly, but I thought it would be reasonable to make a sticky header for all table questions instead.